pkg/compare/path.go

```diff
@@ -51,14 +51,30 @@ func (p Path) Pop() {
 	}
 }
 
-// Contains returns true if the supplied string appears within the Path
+// TODO: case sensitivity?
```

Comment:
should we keep case-sensitive comparison?

Reply:
I think so, yes. We expect case-sensitivity in all Go fields and I'd expect that these strings are reflections of Go fields.

Reply:
Ok, removed TODO comment